package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --open=true",
```

Comment:
This will open localhost 4200 automagically when you use npm start, or yarn start